src/lib/services/OutputProcessor.ts

```diff
@@ -13,6 +14,26 @@ const MESSAGE_END_TAG = 'SFDX-END';
 const REALTIME_MESSAGE_START_TAG = 'SFCA-REALTIME-START';
 const REALTIME_MESSAGE_END_TAG = 'SFCA-REALTIME-END';
 
+/**
+ * A namespace for UUID generation. Randomly generated by https://www.uuidgenerator.net.
+ */
+const UUID_NAMESPACE = '03bcf6c5-7828-4b6b-bc12-94a4bc4ab2f8';
+/**
+ * A universally unique ID for this specific CLI execution.
+ * Will be attached to all internal telemetry events, allowing us to
+ * aggregate all the events in individual runs and spot patterns.
+ */
+const UUID: string = uuidv5('Internal Telemetry', UUID_NAMESPACE);
+
+type InternalTelemetryData = {
```

Comment:
I know that we want to include information about the vertex, but I'm uncertain if that information can be easily retrieved from the point where the events are posted. So for now, I'm going to omit that property, and we can add it if doing so is viable.

Reply:
Makes sense. Can we have a generic hold-anything kind of a property that we can reprovision?

Reply:
We could. I'll do that.